### PR TITLE
Unit tests added for sonar issue fix

### PR DESCRIPTION
**Pull request by btensay**
This branch is to fix sonar issues raised in the pr [PR-2014](https://build.platform.hmcts.net/job/HMCTS_d_to_i/job/ethos-repl-docmosis-service/view/change-requests/job/PR-2014/)

Few unit tests added.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingServiceTest.java [hunk 1]
```diff
@@ -144,6 +145,9 @@ public class ListingServiceTest {
     private ListingDetails listingDetailsRange;
     private DocumentInfo documentInfo;
     private List<SubmitEvent> submitEvents;
+    private static final String TEESSIDE_MAGS = "Teesside Mags";
```
Comment (GauravChawlaMOJ) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingServiceTest.java:148]: reuse them for listingService?

**Review by GauravChawlaMOJ — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingServiceTest.java [hunk 2]
```diff
@@ -144,6 +145,9 @@ public class ListingServiceTest {
...
+    private static final String NEWCASTLE_CFT = "Newcastle CFT";
```
Comment (hsjhita) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingServiceTest.java:149]: did you not create these in ecm-common?

Comment (btensay) [reply src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingServiceTest.java:149]: Good point. I import them.

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingServiceTest.java (hunk 1 above)
Comment (btensay) [reply src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingServiceTest.java]: Fixed

**Review by JustMehmet — APPROVED** [review]

**Pull request merged**